src/Components/UI/NavBar/NavigationButton.styles.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const NavigationButton = styled.button`
```

Comment:
This would nest a `button` inside an `a` in the navigation. There are two possible approaches:

1. use `styled.span` here
2. use `styled.a` here and add `passHref` to `<Link />` in `NavBar.js`